### PR TITLE
Add favicon and title

### LINKED ISSUE #1: Add favicon

### PR DESCRIPTION
**Pull request by benthecarman**
Closes #1

Took the icon from what @suhailsaqan used for lnvortex.com

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -3,7 +3,10 @@ import Link from 'next/link';
 
 export default function Header({ coordinatorName }) {
   return (
+    <>
+    <title> {`Vortex | ${coordinatorName}`} </title>
     <div>
+      <link rel="shortcut icon" href="/favicon.ico" />
```
Comment (deregs) [inline components/Header.tsx]: Can move this line to the `<Head>` section [here](https://github.com/ln-vortex/ln-vortex-gui/blob/ccc23e0a7198bbebeb677a9222b0491e8fe9bedf/pages/_document.tsx#L12-L20)

File: components/Header.tsx [hunk 2]
```diff
@@ -3,7 +3,10 @@ import Link from 'next/link';
 
 export default function Header({ coordinatorName }) {
   return (
+    <>
+    <title> {`Vortex | ${coordinatorName}`} </title>
```
Comment (deregs) [inline components/Header.tsx]: To make sure this goes in the `head` of the page you can `import Head from 'next/head'` and wrap it in a `Head` component.

File: components/Header.tsx (hunk 1 above)
Comment (benthecarman) [reply components/Header.tsx]: I swear I tried that and it didn't work but seems to be working now

File: components/Header.tsx (hunk 2 above)
Comment (benthecarman) [reply components/Header.tsx]: Done

**Review by deregs — APPROVED** [review]

**Pull request merged**